src/pybind/mgr/orchestrator.py

```diff
@@ -835,11 +866,15 @@ def _orchestrator_wait(self, completions):
 
         Waits for writes to be *persistent* but not *effective*.
         """
+        for c in completions:
+            self._update_completion_progress(c)
         while not self.wait(completions):
             if any(c.should_wait for c in completions):
                 time.sleep(5)
             else:
                 break
+        for c in completions:
+            self._update_completion_progress(c, 1)
```

Comment:
if we use the change that i have suggested in _update_completions_progress, you can just call:
**self._update_completion_progress(c)**

Reply:
Fixed!